daemon/src/engine/sys/sdl_input.cpp

```diff
@@ -561,31 +561,56 @@ static keyNum_t IN_TranslateSDLToQ3Key( SDL_Keysym *keysym, bool down )
 	return key;
 }
 
+
+// Whether the cursor is enabled
+static MouseMode mouse_mode = MouseMode::SystemCursor;
+
 /*
- * When this is true, the cursor is free to move and cgame will receive
- * mouse coordinates.
- * When this is false, the cursor will be locked in the window and cgame
- * will receive deltas
+ * Returns whether the cursor is enabled
  */
-static bool cursor_active = true;
+MouseMode IN_GetMouseMode()
+{
+    return mouse_mode;
+}
 
 /*
- * Activates the cursor, the system cursor will still be disabled and
- * the game is responsible of drawing the cursor image
+ * Enables or disables the cursor
  */
-void IN_SetCursorActive( bool active )
+void IN_SetMouseMode(MouseMode new_mode)
```

Comment:
Perfer camelcase for variable names for consistency with the other variables in the file.